components/concept/engine/src/main/java/mozilla/components/concept/engine/prompt/PromptRequest.kt

```diff
@@ -67,6 +66,18 @@ sealed class PromptRequest {
         val onStay: () -> Unit
     ) : PromptRequest()
 
+    /**
+     * Value type that represents a request for a select credit card prompt.
+     * @property creditCards a list of [CreditCard]s to select from.
+     * @property onDismiss callback to let the page know the user dismissed the dialog.
+     * @property onConfirm callback that is called when the user confirms the credit card selection.
+     */
+    data class SelectCreditCardPrompt(
```

Comment:
nit: Maybe we could remove the 'Prompt' part as this will read `PromptRequest.SelectCreditCardPrompt` instead of `PromptRequest.SelectCreditCard`.